### DISCUSSION TITLE
Skipping Local Deployment

### DISCUSSION THREAD

**Discussion by drewcodes0001** [chtrembl/azure-cloud#59] [Q&A] [answered]
Hi, just wanted to ask this question while I'm not too far into the project, only on step 05 right now. I skipped local deployment, so when I visit my app service url the webpage looks like this: 

![Capture](https://github.com/user-attachments/assets/2272cbc0-2349-4915-b392-d6541debf865)

This is how it should look because I skipped deploying locally, correct?

**Comment by chtrembl** [comment, marked as answer]
That looks right for AKS with NGINX deployed (AKS/NGINX are used to the host and route the 3 services that the html web app in the earlier step 2 will point to.) Once you finish step 5 you should be able to curl the 3 services and request json. Happy Learning!
